src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,25 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
-    userCountries.forEach(function(countryName) {
-        var countryTotal = userData[countryName].length;
-
-        countriesArr.push([countryName, countryTotal]);
-        total += countryTotal;
-    });
-
+    var countryTotal = 0;
+    var date = userData.lastUpdated;
+    for (key in userData.institutes) {
```

Comment:
@wkurniawan07 here the objects is number of countries( around 82 ).But my point is not that, even after using the `object.keys` i still have to iterate over it to get the `total`. So whats the point of using it when you can iterate over it directly using `userData.institutes` ?